app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,50 @@
+$( "a.upvote i" ).click(function( event ) {
+  event.preventDefault();
+  event.stopPropagation();
+  $(this).toggleClass("gold");
+  var reviewId = $(this).attr("id").match(/review-([0-9]+)/)[1];
```

Comment:
You want to handle DOM stuff using DOM manipulation instead of data structure manipulation (which might change) later on. You probably can rewrite this with js instead:

`.getElementById(id name).innerHTML`

https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById